mesonbuild/wrap/wrap.py

```diff
@@ -525,8 +543,24 @@ def get_data(self, urlstring: str) -> T.Tuple[str, str]:
         elif WHITELIST_SUBDOMAIN in urlstring:
             raise WrapException(f'{urlstring} may be a WrapDB-impersonating URL')
         else:
+            headers = {}
+            is_http_protocol = url.scheme in ('http', 'https')
+            creds = self.get_netrc_credentials(url.netloc)
+
+            if is_http_protocol:
+                headers.update({'User-Agent': f'mesonbuild/{coredata.version}'})
```

Comment:
Is it a problem if, for simplicity, we init the headers with this rather than only send it for http?